src/components/auth/LoginButton/KakaoLogin.tsx

```diff
@@ -7,11 +7,19 @@ const cx = classNames.bind(styles);
 
 export default function KakaoLogin() {
   return (
-    <a href={API_BASE_URL + '/auth/kakao'}>
-      <div className={cx('kakaoButton')}>
-        <KakaoLogo />
-        <span>카카오로 계속하기</span>
-      </div>
-    </a>
+    <>
+      <a href={API_BASE_URL + '/auth/kakao/server'}>
+        <div className={cx('kakaoButton')}>
+          <KakaoLogo />
+          <span>카카오로 계속하기(서버)</span>
+        </div>
+      </a>
+      <a href={API_BASE_URL + '/auth/kakao/local'}>
+        <div className={cx('kakaoButton')}>
+          <KakaoLogo />
+          <span>카카오로 계속하기(로컬)</span>
+        </div>
+      </a>
```

Comment:
❗서버랑 로컬이랑 분기가 필요한 경우에는 `.env` 활용하면 어떨까 싶어요!

변수명은 제가 `NEXT_PUBLIC_AUTH_END_POINT`로 했지만 좋은 네이밍으로 변경해 주시면 될 거 같습니다!

[로컬 서버 env]
```
NEXT_PUBLIC_AUTH_END_POINT=local
```
[배포 서버 env]
```
NEXT_PUBLIC_AUTH_END_POINT=server
```
```suggestion
      <a href={API_BASE_URL + `/auth/kakao/${process.env.NEXT_PUBLIC_AUTH_END_POINT}`>
        <div className={cx('kakaoButton')}>
          <KakaoLogo />
          <span>카카오로 계속하기</span>
        </div>
      </a>
```

Reply:
반영했습니다